### ISSUE TITLE
F12 key does not work with FN Lock in GHelper

### ISSUE THREAD

**Issue by CodingGenius0001** [seerge/g-helper#2995]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [x] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

When the FN Lock is enabled, all function keys work except the F12 key until and unless I hold the FN key along with pressing the F12 button

---

Basically:
1. Pressing F12 key alone (with FN-Lock on in GHelper) = registers the F12 key instead of Airplane Mode (in my case)
2. Pressing F12 key + FN key (still with FN-Lock on in GHelper) = registers Airplane Mode 

### How to reproduce the bug?

1. Enable FN-Lock above extra button
2. Press F12

### Logs

[log.txt](https://github.com/user-attachments/files/16656085/log.txt)

### Device and Model

Asus Zephyrus G14 2024 GA403UI

### Armoury Crate

Uninstalled

### Asus Services

0

### Version

0.184.0

### OS

Windows 11 23H2

**Comment by seerge** [comment]
@CodingGenius0001 this is a dublicate https://github.com/seerge/g-helper/issues/2822
